optimizely/interface.go

```diff
@@ -31,6 +31,7 @@ type ProjectConfig interface {
 	GetEventByKey(string) (entities.Event, error)
 	GetExperimentByKey(string) (entities.Experiment, error)
 	GetFeatureByKey(string) (entities.Feature, error)
+	GetFeatureMap() map[string]entities.Feature
```

Comment:
Hmm, this is a proposed public API. I think what is more relevant is that we get the list of all features, rather than just the map. So let's expose a `GetFeatureList` API instead and use that in the client. @mjc1283 FYI